Resources/Prototypes/Entities/Clothing/Shoes/boots.yml

```diff
@@ -78,14 +78,97 @@
     sprite: Clothing/Shoes/Boots/laceups.rsi
 
 - type: entity
-  parent: ClothingShoesBaseButcherable
+  parent: ClothingShoesBaseWinterBoots
   id: ClothingShoesBootsWinter
   name: winter boots
-  description: Fluffy boots to help survive even the coldest of winters.
   components:
   - type: Sprite
     sprite: Clothing/Shoes/Boots/winterboots.rsi
   - type: Clothing
     sprite: Clothing/Shoes/Boots/winterboots.rsi
-  - type: TemperatureProtection
-    coefficient: 0.2
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterCargo
+  name: cargo winter boots
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/winterbootscargo.rsi
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/winterbootscargo.rsi
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterEngi
+  name: engineering winter boots
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/winterbootsengi.rsi
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/winterbootsengi.rsi
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterMed
+  name: medical winter boots
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/winterbootsmed.rsi
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/winterbootsmed.rsi
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterSci
+  name: science winter boots
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/winterbootssci.rsi
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/winterbootssci.rsi
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterSec
+  name: security winter boots
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/winterbootssec.rsi
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/winterbootssec.rsi
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterSyndicate
+  name: syndicate's winter boots
```

Comment:
the syndie winter coat has a custom description so this have one as well based on the coat

Reply:
Like this?